projects/f3d.suffix.cmake

```diff
@@ -0,0 +1,21 @@
+# Set suffix to be used for generating archives. This ensures that the package
+# files have decent names that we can directly upload to the release page.
+set(package_suffix_items)
+
+if (ospray_enabled)
+  list(APPEND package_suffix_items
+    "raytracing")
+endif()
+
+if (APPLE)
+  list(APPEND package_suffix_items
+    "macOS")
```

Comment:
What is the `CMAKE_SYSTEM_NAME` value for macOS?

Reply:
Darwin 🤦